etc/testing/circle/install.sh

```diff
@@ -16,6 +16,7 @@ sudo apt-get install -y -qq \
   conntrack \
   pv \
   shellcheck \
+  moreutils \
```

Comment:
what's in here?

Reply:
That "ts" utility that prints a timestamp after each line.  `echo -e 'foo\nbar' | ts` will print 
```
[May 4 2:18PM] foo
[May 4 2:18PM] bar
``` 
for example.

This basically lets us see the time that each test log line was produced (same for the k8s events we log now) and correlate them.  It's not a fun process to correlate them, but it is at least possible now :)